datastacks/utils.py

```diff
@@ -76,6 +77,16 @@ def generate_pipeline(
     template_source_path = f"de_templates/{stage_name}/{template_source_folder}/"
     target_dir = generate_target_dir(stage_name, config.dataset_name)
 
+    if Path(f"{target_dir}").exists():
```

Comment:
I haven't tested it, so I'm not sure, but what is the behaviour when we are generating a pipeline with Data Quality? Does this mean that a user would have to confirm overwriting for DQ files?

Reply:
![image](https://github.com/amido/stacks-azure-data/assets/131858636/4ceb913b-7bf4-4be6-93ef-d82645b68938)
Nope 😄

Reply:
oh, yeah, I see, it's handled elsewhere, thanks!